### PR TITLE
Fix JS error on /external-site

### PR DESCRIPTION
**Pull request by virginiacc**
The header & footer scripts were initializing on /external-site even though it had no header or footer, which caused a JS error

## Changes

- Build scripts for external site separately

## Testing

- Check out branch & run gulp
- Navigate to http://localhost:8000/external-site/ and check that there is no js error related to missing header element

## Review

- @sebworks 
- @contolini 

## Screenshots

screen shot 2016-08-23 at 8 26 45 am

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/external-site/index.html [hunk 1]
```diff
@@ -31,6 +31,12 @@
     {# No footer needed on this page. #}
 {%- endblock %}
 
+{% block javascript %}
+<script type='text/javascript'>
+    {% include '/js/externalSite.js' %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/external-site/index.html]: TRIVIAL: should probably be `external-site.js` to follow naming style of other non-class modules in the project.

**Comment by anselmbradford** [comment]
Fixes https://github.com/cfpb/cfgov-refresh/issues/1838

**Comment by sebworks** [comment]
👍 , @anselmbradford please sign off on this as well.

File: cfgov/jinja2/v1/external-site/index.html (hunk 1 above)
Comment (virginiacc) [inline cfgov/jinja2/v1/external-site/index.html]: Good point, @anselmbradford -- fixed in cb8044c

**Comment by contolini** [comment]
👍

**Comment by anselmbradford** [comment]
Works as described! 👍

**Pull request merged**